CMakeLists.txt

```diff
@@ -164,6 +164,12 @@ if(HAVE_LIBEXECINFO)
     target_link_libraries(Hyprland execinfo)
 endif()
 
+check_include_file("sys/timerfd.h" HAS_TIMERFD)
+pkg_check_modules(epoll IMPORTED_TARGET epoll-shim)
+if(NOT HAS_TIMERFD AND epoll_FOUND)
```

Comment:
but Hyprland hard-includes `<sys/timerfd.h>`, will epoll magically add that header?

Reply:
Yes:
```
$ pkg info -xl epoll-shim | fgrep timerfd
        /usr/local/include/libepoll-shim/sys/timerfd.h
$ pkg-config --cflags epoll-shim
-I/usr/local/include/libepoll-shim
```

Reply:
cool